lib/bot.js

```diff
@@ -33,6 +34,25 @@ class Bot {
     this.ircNickColor = options.ircNickColor !== false; // default to true
     this.channels = _.values(options.channelMapping);
 
+    this.formatCommandPrelude = options.formatCommandPrelude || 'Command sent from Discord by {$nickname}:';
+    this.formatIRCText = options.formatIRCText || '<{$displayUsername}> {$text}';
+    this.formatURLAttachment = options.formatURLAttachment || '<{$displayUsername}> {$attachmentURL}';
+    // "{$keyName}" => "variableValue"
```

Comment:
Nitpick, but I think it makes more sense to put these comments above the variables.